tests/trestle/core/utils_test.py

```diff
@@ -218,8 +211,8 @@ def test_parameter_to_dict() -> None:
     test1 = common.Test(expression='not too big', remarks='test for 1')
     test2 = common.Test(expression='keep it small', remarks='test for 2')
     constraints = [common.ParameterConstraint(description='my constraints', tests=[test1, test2])]
-    sel = common.ParameterSelection(how_many=common.HowMany.one_or_more, choice=['one', 'two', 'three'])
-    values = [common.ParameterValue(__root__='one'), common.ParameterValue(__root__='two')]
+    sel = common.ParameterSelection(how_many='one-or-more', choice=['one', 'two', 'three'])
```

Comment:
can use common.const.ONE_OR_MORE instead of string here

Reply:
changed all such to const.ONE_OR_MORE_HYPHENED